packages/cli/package.json

```diff
@@ -27,9 +27,9 @@
     "prebuild": "npm run clean",
     "build": "tsc -b",
     "postpack": "shx rm -f oclif.manifest.json",
-    "prepack": "npm run build && oclif manifest && oclif readme",
+    "prepack": "npm run build && oclif manifest && npm run readme",
     "test": "jest",
-    "version": "oclif readme && git add README.md"
+    "readme": "oclif readme && shx sed -i \"s/^_See code:.*$//g\" README.md"
```

Comment:
Worth upstreaming a fix here in case its small?

Reply:
Related to this bug or something more specific around monorepos? https://github.com/oclif/oclif/issues/1128

Reply:
I poked at the code a bit and it wasn't obvious how to resolve. Will log a bug.